### PR TITLE
gh-926 Display rejection transaction

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/UI/Transaction/TransactionListViewController/Cells/TransactionListTableViewCell.swift [hunk 1]
```diff
@@ -23,6 +23,8 @@ class TransactionListTableViewCell: SwiftUITableViewCell {
     @IBOutlet private weak var bottomStackView: UIStackView!
     @IBOutlet private weak var confirmationsCountLabel: UILabel!
     @IBOutlet private weak var confirmationsCountImageView: UIImageView!
+    @IBOutlet weak var highlightBarView: UIView!
```
Comment (sche) [inline Multisig/UI/Transaction/TransactionListViewController/Cells/TransactionListTableViewCell.swift]: can we use private here to be the same style?

**Review by sche — APPROVED** [review]

File: Multisig/UI/Transaction/TransactionListViewController/QueuedTransactionsViewController.swift [hunk 2]
```diff
@@ -47,6 +47,15 @@ class QueuedTransactionsViewController: TransactionListViewController {
         date.timeAgo()
     }
 
+    override func shouldHighlight(transaction: SCGModels.TxSummary) -> Bool {
+        switch transaction.txInfo {
+        case .rejection(_):
+            return true
```
Comment (DmitryBespalov) [inline Multisig/UI/Transaction/TransactionListViewController/QueuedTransactionsViewController.swift:53]: The `transaction.txInfo == .rejection` doesn't work, does it?

**Review by DmitryBespalov — APPROVED** [review]
small comments

File: Multisig/UI/Transaction/TransactionListViewController/QueuedTransactionsViewController.swift (hunk 2 above)
Comment (MouazAlzahabi) [reply Multisig/UI/Transaction/TransactionListViewController/QueuedTransactionsViewController.swift:53]: Nope unfortunately

**Pull request merged**